test/functional/load-balancer-spec.test.js

```diff
@@ -15,17 +15,15 @@ const SKIP = [
   //       in sub-optimal pinning.
   'pinned connections are not returned after an network error during getMore',
   'pinned connections are not returned to the pool after a non-network error on getMore',
-  'stale errors are ignored'
+  'stale errors are ignored',
+  // NOTE: The driver correctly fails these 2 tests in non LB mode due to validation that
+  // loadBalanced=true is only valid as a URI option and not a regular driver option according
+  // to the spec. In versions that are 3.4 or less an error still occurs but a different one (connection closes).
+  // TODO(NODE-3543): fix the path-ing that will produce errors for older servers
```

Comment:
Brought back the skip and made a ticket for the issue.

Reply:
can we remove the no longer true bit of the note concerning the URI option parsing?

Reply:
done!